databrowse/plugins/db_PDF_viewer/db_PDF_viewer.py

```diff
@@ -108,17 +123,24 @@ def getContent(self):
                     xmlchild.text = self.ConvertUserFriendlyPermissions(st[ST_MODE])
 
                     # User and Group
-                    username = pwd.getpwuid(st[ST_UID])[0]
-                    groupname = grp.getgrgid(st[ST_GID])[0]
-                    xmlchild = etree.SubElement(xmlroot, "owner", nsmap=self.nsmap)
-                    xmlchild.text = "%s:%s" % (username, groupname)
+                    if platform.system() == "Linux":
```

Comment:
Code like this is repeated in a variety of places in many of the plugins.  We should move this into a single centralized location.  This probably doesn't need to be done now, but it needs to be done.

Reply:
You mean the file statistics?